src/mds/main/main.cpp

```diff
@@ -102,27 +103,27 @@ int main(int argc, char **argv) {
         }
     }
 
-    // 初始化日志模块
+    // initialize logging module
     google::InitGoogleLogging(argv[0]);
 
     curve::mds::MDS mds;
 
-    // 初始化各个选项
+    // initialize MDS options
     mds.InitMdsOptions(conf);
 
-    // 启动 MDS DummyServer用于主从MDS探活及Metric导出
+    // start MDS dummy server for master-slave liveness probe and Metric exportation of MDS //NOLINT
     mds.StartDummy();
 
-    // Master 竞选
+    // leader election (for leader MDS)
     mds.StartCompaginLeader();
 
-    // 选举成功进行初始化所有模块
+    // initialize other modules after master selection
     mds.Init();
 
-    // 启动对外服务
+    // start mds service
```

Comment:
start mds server，and wait CTRL+C to quit

Reply:
> start mds server，and wait CTRL+C to quit

done